### DISCUSSION TITLE
Version 2.0.1

### DISCUSSION THREAD

**Discussion by sonia-devprose** [secure-dev-ops/code-realtime#29] [General]
📢 New version of DevOps Code RealTime - 2.0.1 has been published in [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=secure-dev-ops.code-realtime-ce), and [Open VSX Registry](https://open-vsx.org/extension/secure-dev-ops/code-realtime-ce)

🚀 DevOps Code RealTime 2.0.1 introduces enhanced C++ code generation with trigger guard support for both capsule and class state machines. Developers can now leverage the rule_config property to selectively disable validation rules on capsule parts, entry, and exit points, improving flexibility during debugging. The release also features a new Quick Fix to easily disable problematic validation rules on elements. Support for unconnected service ports allows event injection via the Art Debugger. Additional improvements include consistent handling of outgoing transitions for choices, new samples like TrafficLightWeb for Node.js web interfaces, and enhancements to the Art Debugger for injecting events with directional indicators. Check out the full release notes for more details!

Check the release notes [here](https://secure-dev-ops.github.io/code-realtime/releases/CHANGELOG/).

👁️‍🗨️ Stay tuned for further releases and do not forget to share your [thoughts](https://github.com/secure-dev-ops/code-realtime/discussions).

**Comment by Sonia-bit** [comment]
Closing as the case is duplicate
